includes/EmailNotification.php

```diff
@@ -233,11 +245,6 @@ private function canSendUserTalkEmail() {
 			return false;
 		}
 
-		// Is it a minor edit? If so, does that user want to know about minor edits?
-		if ( $this->isMinorEdit() &&  $targetUser->getOption( 'enotifminoredits' ) )  {
```

Comment:
Notice this logic changed slightly when it moved up above.  The logic on this line is incorrect because it says "If this is a minor edit AND this user wants to get minor edits THEN don't send an email."